subprojects/tooling-api/src/main/java/org/gradle/tooling/events/transform/package-info.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Artifact transform execution specific interfaces and classes related to event notifications.
+ */
+package org.gradle.tooling.events.transform;
```

Comment:
Should this have `@NonNullApi`?

Reply:
We don't use that for the other TAPI packages. Would make sense to add, though, I think.

Reply:
Sounds useful. Let's capture that in a separate issue then.